hoomd/ComputeThermo.cc

```diff
@@ -416,6 +416,16 @@ void export_ComputeThermo(py::module& m)
     .def(py::init< std::shared_ptr<SystemDefinition>,std::shared_ptr<ParticleGroup>,const std::string& >())
     .def("getTemperature", &ComputeThermo::getTemperature)
     .def("getPressure", &ComputeThermo::getPressure)
+    .def("getPressureXX", &ComputeThermo::getPressureXX)
+    .def("getPressureXY", &ComputeThermo::getPressureXY)
+    .def("getPressureXZ", &ComputeThermo::getPressureXZ)
+    .def("getPressureYY", &ComputeThermo::getPressureYY)
+    .def("getPressureYZ", &ComputeThermo::getPressureYZ)
+    .def("getPressureZZ", &ComputeThermo::getPressureZZ)
+    .def("getNDOF", &ComputeThermo::getNDOF)
+    .def("getTranslationalDOF", &ComputeThermo::getTranslationalDOF)
+    .def("getRotationalDOF", &ComputeThermo::getRotationalDOF)
+    .def("getNumParticles", &ComputeThermo::getNumParticles)
```

Comment:
We could make some of these pybind properties so we don't have to use `()` everywhere. That may be a bit cleaner as all of these are simple quantities that don't take arguments.

Reply:
I don't have a strong opinion either way. Once I make the changes for the pressure tensor, there will be one quantity that is not simple anymore.